### ISSUE TITLE
Add nginx as an Option

### ISSUE THREAD

**Issue by gordonbanderson** [silverstripe/silverstripe-routewhitelist#9]
I prefer to use nginx rather than Apache.  It would be good if suitable config was generated optionally for nginx also

**Comment by gordonbanderson** [comment]
I have config generating based on flags in the config file but it appears to not be the right config,as in it doesn't work :)  Will have a look with fresher eyes tomorrow.

**Comment by candidasa** [comment]
Any update on the nginx config?

**Comment by gordonbanderson** [comment]
Nope, other than it broke a personal site for a while after a reboot.  I've not had time to have another go at it.
